pkg/api/controller.go

```diff
@@ -2272,6 +2272,23 @@ func (c *Controller) SetGarbageCollectionRules(w http.ResponseWriter, r *http.Re
 	writeResponse(w, http.StatusNoContent, nil)
 }
 
+func (c *Controller) DeleteGarbageCollectionRules(w http.ResponseWriter, r *http.Request, repository string) {
+	if !c.authorize(w, r, permissions.Node{
+		Permission: permissions.Permission{
+			Action:   permissions.SetGarbageCollectionRulesAction,
+			Resource: permissions.RepoArn(repository),
+		},
+	}) {
+		return
+	}
+	ctx := r.Context()
+	err := c.Catalog.SetGarbageCollectionRules(ctx, repository, nil)
```

Comment:
What does it translate to? is it saving an empty json?

Reply:
It saves an empty file (not an empty json)